src/client/app/containers/CompareChartContainer.ts

```diff
@@ -84,6 +87,18 @@ function mapStateToProps(state: RootState, ownProps: CompareChartContainerProps)
 		}
 	}
 
+	// To Do: When I click this icon it crashes OED. The error relates to using a Hook (useState, I think) 
```

Comment:
This should be TODO (instead of To Do). That way it shows up at a TODO item in many IDEs. Same to To Do below.

Really small: There is an extra space at the end of the line on this and the next line. The linting does not like this. This is why the tests are failing on GitHub.